src/prefect/client/orchestration.py

```diff
@@ -247,6 +255,30 @@ def api_url(self) -> httpx.URL:
         """
         return self._client.base_url
 
+    async def events(self) -> EventsClient:
+        if not self._events_client:
+            if (
+                experiment_enabled("events_client")
+                and self.server_type == ServerType.CLOUD
+            ):
+                self._events_client = PrefectCloudEventsClient(
+                    api_url=PREFECT_CLOUD_API_URL.value(),
+                    api_key=prefect.settings.PREFECT_API_KEY.value(),
+                )
+            else:
+                self._events_client = NullEventsClient()
+
+            await self._events_client.__aenter__()
+
+        return self._events_client
```

Comment:
I notice that the `PrefectClient` has an `_exit_stack`, is this something we could join in on as well?

Reply:
Possibly, I'll look at it.